internal/config/config.go

```diff
@@ -38,7 +38,7 @@ type Config struct {
 
 var globalConfig = &Config{
 	BindAddr:      "0.0.0.0",
-	CardanoConfig: "./configs/cardano/preview/config.json",
+	CardanoConfig: "./config/cardano/preview/config.json",
```

Comment:
This path doesn't exist. We could use `./config/cardano/testdata/config.json` here, but that feels kinda wrong

Reply:
I really would prefer we embed the default set into the binary, so we function out of the box without additional files. That's the direction I was going with this, but I'll hold off and do it all at once.

Reply:
I cheated and made a symlink, for now. It was a symlink before, so I am calling it fair game.